jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,465 @@
+:toc-title:
+version:
+- Cloud
+
+NOTE: CircleCI runner is currently only available if you are on the Scale Plan. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+
+This page describes how to install CircleCI runner on Linux and macOS platforms.
+
+toc::[]
+
+=== Prerequisites
+
+==== Support Levels
+
+There are currently three support levels for CircleCI runner:
+
+* Supported
+* Untested
+* Unsupported
+
+===== Supported
+
+* CircleCI has built and tested runner on a given platform.
+* Support expectation:
++
+** DCEs can provide support.
+** CircleCI documentation and best practices available.
+** If issues arise, CircleCI will assist the customer to resolve the issue within the usual Gold support Service Level Agreements (SLAs).
+
+===== Untested
+
+* CircleCI has not tested runner on a given platform but it should work based on the similarities of the platform to one or more supported platforms.
+* Support expectation:
++
+** There is no CircleCI documentation available.
+** Customers can run CircleCI runner on an untested platform, but the customer is responsible for determining how to run runner on this platform.
+** If issues arise on the untested platform, DCEs might or might not be able to help.
+** DCEs will provide general guidance (not specific to this platform) on best practices for CircleCI runner.
+** If a customer is able to get an untested platform up and running, DCEs will collect information about their setup in order to reuse information with other customers.
+
+===== Unsupported
+
+* CircleCI will not provide a runner binary capable of running CircleCI runner on the platform.
+* Support expectation:
++
+** CircleCI runner will not work on the platform.
+** A customer can try running CircleCI runner on the platform, but are unlikely to succeed.
+** No CircleCI documentation or DCE support.
+
+==== Supported Platforms
+
+The following platforms have been validated to work with CircleCI runner:
+
+* Intel + Ubuntu
+* Arm64 + Ubuntu
+* Intel + macOS
+
+===== Untested Platforms
+
+The following platforms have not been tested, but are believed to work with CircleCI runner:
+
+* Intel + non-Ubuntu Linux (Debian, Red Hat, SUSE, etc)
+* Arm64 + non-Ubuntu Linux (Debian, Red Hat, SUSE, etc)
+* Docker
+
+NOTE: CircleCI provides no documentation for these platforms.
+
+===== Unsupported Platforms
+
+The following platforms are not currently supported to work with CircleCI runner:
+
+* Intel + Windows
+* Arm + Windows
+* Apple Silicon + macOS
+* Arm32 + Linux
+* Run natively in Kubernetes/Rancher/other container orchestration solutions
+
+==== Authentication
+
+In order to complete this process you will need to create a namespace, https://circleci.com/docs/2.0/optimizations/#resource-class[resource-class] and authentication token by performing the steps listed below:
+
+1. Install the circleci command-line tool.
+2. Create a namespace for your organization's runner resources under:
```

Comment:
Feels like there's something missing at the end of this line?

Reply:
Fixed

Reply:
Fixed